app/mailers/competitions_mailer.rb

```diff
@@ -175,6 +175,10 @@ def registration_reminder(competition, user, registered_but_not_accepted)
     delegates.flat_map { |delegate| delegate.senior_delegates.map(&:email) }.uniq.compact
   end
 
+  private def delegates_to_regional_delegates_email(delegates)
+    delegates.flat_map { |delegate| delegate.region&.lead_user.email }.uniq.compact
```

Comment:
`region` (of `delegate.region`) will soon be removed. User won't have direct relation with the region. But like `def senior_delegates` of `user.rb`, you can create a method for regional delegates as follows:
```
  def regional_delegates
    active_roles
      .select { |role| UserRole.is_group_type?(role, UserGroup.group_types[:delegate_regions]) }
      .map { |role| UserRole.group(role).lead_user }
  end
```

Then you can re-write this line as:
```
delegates.flat_map { |delegate| delegate.regional_delegates.map(&:email) }.uniq.compact
```